R/visualisations.R

```diff
@@ -297,3 +297,61 @@ plot_pmgl_vs_true <- function(dataset,
     return(plt)
   }
 }
+
+#' @title Plot transformed linear GWAS beta values against the LTFH_transformed
+#' values.
+#'
+#' @description description to be written.
+#'
+#' @param dataset data imported to R.
+#' @param transformed_beta the transformed beta values from LTFH.
+#' @param linear_beta the beta values from standard GWAS.
+#' @param bonferroni corrected significant values from LTFH.
+#' @param save_plot_path if \code{FALSE}, return the function returns a ggplot
+#' object. Else a path of the directory to save the plot to.
+#' @param plot_filename name of the file to be saved, including file extension.
+#'
+#' @return Either returns a \code{ggplot} object or saves the plot to
+#' \code{save_plot_path} and returns NULL.
+#'
+#' @export
+GWAS_beta_VS_LTFH_transfored_beta <- function(dataset,
+                                              transformed_beta,
```

Comment:
Is transformed_beta, linear_beta og bonferroni not columns in the dataset?

Reply:
It should be after the mutator function has been applied :). The reason why I made it as input, is becouse you might want to compare other analysis types of beta-values. E.g the beta-values from liab agianst ltfh or linear gwas. But I think it is a better call to change the names of these inputs and the function ;).